.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,39 @@
+name: 🐞 Bug Report
+description: Tell us about something that's not working the way we (probably) intend.
+labels: ["Kind:Bug", "State:Triage"]
```

Comment:
```suggestion
```

In Elixir, everything untagged is untriaged. Maybe having a explicit state is good, but I want to try that within Ecto first. :)